website/docs/plugins/submitit_launcher.md

```diff
@@ -35,23 +35,28 @@ Submitit actually implements 2 different launchers: `submitit_slurm` to run on a
 You can discover the SLURM Launcher parameters with:
```

Comment:
can you fold this block like in the Ray Launcher docs?
see https://hydra.cc/docs/plugins/ray_launcher

Reply:
Done, somehow it turned out to be more complicated than I anticipated due to some formatting quirks of react